plugins/modules/gitlab_project.py

```diff
@@ -172,6 +172,39 @@
       - This option is only used on creation, not for updates. This is also only used if I(initialize_with_readme=true).
     type: str
     version_added: "4.2.0"
+  builds:
+    description:
+      - C(private) Repository CI/CD allowed only to project members.
+      - C(disabled) Repository CI/CD is disabled.
+      - C(enabled) Repository CI/CD is enabled.
+    default: enabled
+    type: str
+    choices: ["private", "disabled", "enabled"]
+    aliases:
+      - builds_access_level
+    version_added: "6.1.0"
```

Comment:
6.1.0 has already been released, you cannot add new options for it. The next release will be 6.2.0:
```suggestion
    version_added: "6.2.0"
```
(same below)